tests/step_implementers/push_artifacts/test_maven_push_artifacts.py

```diff
@@ -383,64 +499,69 @@ def test_push_artifact_with_artifacts_results_multi(self, mvn_mock):
                     'push-artifacts': {
                         'implementer': 'Maven',
                         'config': {
-                            'url': 'http://artifactory.apps.tssc.rht-set.com/artifactory/tssc/',
+                            'maven-push-artifact-repo-id': 'repoid',
+                            'maven-push-artifact-repo-url': 'repourl',
+                            'maven-servers': [
+                                {'username': 'USER1', 'password': 'PW1'},
```

Comment:
@ppremru mentioned updating `username` to `user`...

Reply:
nope... 
the tag-source, push-artifacts, and deploy are have 'username".   
the one that is not following the pattern is static-code-analysis, it has 'user'.  i added a note to NAPSSPO-976 to change this. (976 has sonarqube enhancments...)